src/Dropdown.svelte

```diff
@@ -54,6 +55,7 @@
       <div
         class="dropdown-item"
         class:selected={value === selected}
+        use:twemoji
```

Comment:
Where do we use emojis in dropdowns?

Reply:
For example the `SeedDropdown` can use emojis, once used
`const key = `${seed.emoji} ${seed.host}`;`